.github/workflows/josh-mongo.yml

```
@@ -35,6 +35,5 @@ jobs:
         with:
           mongodb-version: 4.2
       - run: npm i -g yarn
-      - run: cd josh-sqlite; yarn; yarn test; yarn lint;
-      - run: cd josh-mongo; yarn; yarn test; yarn lint
-      - run: cd josh-json; yarn; yarn test; yarn lint
+      - run: cd josh-tests; yarn;
+      - run: cd josh-mongo; yarn; yarn test; yarn lint;
```

```suggestion
      - run: cd josh-tests && yarn
      - run: cd josh-mongo && yarn --ignore-scripts --frozen-lockfile && yarn test && yarn lint;
```
Same as comment above.